pkg/devfile/adapters/kubernetes/component/adapter_test.go

```diff
@@ -266,7 +277,7 @@ func TestDoesComponentExist(t *testing.T) {
 			}
 
 			// Verify that a component with the specified name exists
-			componentExists := componentAdapter.DoesComponentExist(tt.getComponentName)
+			componentExists, _ := componentAdapter.DoesComponentExist(tt.getComponentName)
```

Comment:
we should check the error here.

Reply:
explicitly returning nil like I mentioned here https://github.com/openshift/odo/pull/3405/files#r446364162

Reply:
https://github.com/openshift/odo/pull/3405#discussion_r446839977 will solve this problem